modules/ppcp-button/resources/js/modules/Helper/CartJanitor.js

```diff
@@ -0,0 +1,65 @@
+class CartJanitor {
```

Comment:
Why `CartJanitor` name? if there is not a really needed reason to use it I would rather prefer a standard name like "CartHelper" or similar here.

Reply:
It was dedicated to only clean the cart if necessary. But ok we may make it more general, changed the name.